### ISSUE TITLE
Boundary clipping truncates surrounding features in PT 3.0

### ISSUE THREAD

**Issue by tpcolson** [nationalparkservice/places-data#266]
Boundary clipping truncates surrounding features which are necessary for visitor navigation experience. 
http://insidemaps.nps.gov/park-tiles-3/#17/35.71454/-83.52488 
![image](https://cloud.githubusercontent.com/assets/5861784/11398120/9dfc19d0-934c-11e5-8e73-968007483cb5.png)
http://insidemaps.nps.gov/park-tiles-3/#17/35.72037/-83.64175
![image](https://cloud.githubusercontent.com/assets/5861784/11398132/b9d7bd4e-934c-11e5-9a40-196e14707a97.png)
http://insidemaps.nps.gov/park-tiles-3/#16/35.7161/-83.6699
![image](https://cloud.githubusercontent.com/assets/5861784/11398151/ce91db98-934c-11e5-9e19-92885c5c52b8.png)
http://insidemaps.nps.gov/park-tiles-3/#16/35.7174/-83.8224
![image](https://cloud.githubusercontent.com/assets/5861784/11398159/dc003b6c-934c-11e5-9b7e-6665ca228b0f.png)
http://insidemaps.nps.gov/park-tiles-3/#17/35.77794/-83.11546
![image](https://cloud.githubusercontent.com/assets/5861784/11398177/f8b55008-934c-11e5-8bf9-7c4f366b1752.png)

**Comment by chadlawlis** [comment]
These surrounding features are pulling from OpenStreetMap, clipped at the park boundary, while any data inside the boundary is pulling from Places. So, these roads will have to be drawn inside the park boundary in Places to show up in Park Tiles. Features can also be added outside the park boundary, and we are in the process of determining how best to handle the rendering (whether to clip OSM data at the unit boundary, at a buffer around the unit boundary, or at another preset "area of interest" bounding box).

The label for "US 321" showing up in the second image is a labeling/cartography issue, however, as it appears to be pulling from the OSM road. cc @mtaylorlong

**Comment by tpcolson** [comment]
That was my initial idea to solve this, BUT, won't that make those road [segments] available for download via the CartoDB download/extract? That would violate a few licensing agreements, in general, a NPS unit can't "publish" source GIS data outside of it's boundaries unless there's a MOA/U in place. If you can somehow tag these cartographic-only elements as "Use in PT only, DO NOT make them available for public download as GIS data", then that's a workable solution, albeit one with a lot of work.

**Comment by tpcolson** [comment]
Further thought..another solution would be to cook up a stand-alone table of OSM way id's, and hook that back to Tile Mill, and say "Use these Way ID's when tiling even though they're outside a unit boundary".

**Comment by chadlawlis** [comment]
Yes, any road segments outside a unit boundary will be available for download given they are associated with a unit code. The boundary issue in general is a political one, while I am not familiar with the need for a MOU for publishing data outside a given unit boundary. Parks and regions have been requesting this directly, given some visitor centers and other important park-owned/associated features fall outside the "official" management boundaries in some cases, so this should have us covered. cc @nateirwin 

The concern with referencing OSM data in any way is the licensing, which prohibits us from aligning our Places data inside the park with the intersecting OSM data outside the park. The idea you propose doesn't seem to violate that, but something to keep in mind.

**Comment by tpcolson** [comment]
I understand that publishing park (federally-owned) assets that fall outside of unit boundaries is feasible, but what about the issue depicted in the screen shots?

**Comment by mtaylorlong** [comment]
Lot's of insightful comments here @tpcolson. 

@cwlawlis802 is right that the US 321 is coming from OSM. We've included these labels on top of the parks thus far so we don't have to have special US highway labels for road segments drawn within park boundaries, but we may be able to just get rid of this practice. I'll have to think a bit more about how that will play out in more urban areas.

Regarding your comments about publishing GIS data outside of park boundaries, it sounds like we're going to have to think carefully about how we "brand" this data. While much of the data in Park Tiles does originate from GIS sources, it's really Cartographic data, which may be less precise than GIS data and may include features that wouldn't show up in traditional gis data, like features just for labeling, or roads that NPS doesn't maintain but still show up inside the park on the basemap since they are part of the visitor experience.

Still, you don't necessarily have to draw roads outside of your park boundary. To fix the missing roads from your screenshots, we would recommend you draw in those roads where they fall within the park boundary. Keep in mind that we can't use the OSM roads at all - even to look at while digitizing. However, we can and should digitize them over imagery if they fall within the boundary. It's up to us at NPMap to make the transition from inside the boundary (places) to outside the boundary (osm) make sense visually.

These are great things to be discussing – let's keep this up!

**Comment by mtaylorlong** [comment]
I've resolved one of the above issues for the next update of PT3 Beta: OSM highway shield will no longer show up inside parks. My next goal is to implement our own highway shields for highways in NPS Places, but it might not happen in this release.

Many of the other issues discussed above can still be resolved by drawing in the missing road segments that fall within park boundaries. Basically, if it falls within park boundaries it has to be in places to show up. I see the issue posed by drawing a bunch of data outside of park boundaries, but that's not necessary to fill in the missing segments highlighted above.
